cypress/integration/EditAndBranching/TagAndFrog.spec.js

```diff
@@ -8,34 +8,46 @@ import * as modelPage from '../../support/components/ModelPage'
 import * as scorerModal from '../../support/components/ScorerModal'
 import * as train from '../../support/Train'
 import * as editDialogModal from '../../support/components/EditDialogModal'
+import * as helpers from '../../support/Helpers'
 
-describe('EditAndBranching', () => {
-  it('Tag And Frog', () => {
-    const textEntityPairs = [{text: 'Tag', entity: 'multi'}, {text: 'Frog', entity: 'multi'}]
+describe('Tag and Frog - Edit And Branching', () => {
+  const textEntityPairs = [{text: 'Tag', entity: 'multi'}, {text: 'Frog', entity: 'multi'}]
 
+  afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
+  
+  context('Setup', () => {
+    it('Should import a model and wait for training to complete', () => {
     models.ImportModel('z-tagAndFrog2', 'z-tagAndFrog2.cl')
     modelPage.NavigateToTrainDialogs()
     cy.WaitForTrainingStatusCompleted()
+    })
+  })
 
-    train.EditTraining('This is Tag.', 'This is Tag.', 'Hi')
-    editDialogModal.SelectChatTurnExactMatch('This is Tag.')
+  context('Edit Train Dialog', () => {
+    it('Should remove two entity labels from alternative input', () => {
+      train.EditTraining('This is Tag.', 'This is Tag.', 'Hi')
+      editDialogModal.SelectChatTurnExactMatch('This is Tag.')
 
-    editDialogModal.VerifyEntityLabelWithinSpecificInput([textEntityPairs[0]], 0)
-    editDialogModal.VerifyEntityLabelWithinSpecificInput(textEntityPairs, 1)
-    editDialogModal.VerifyEntityLabelWithinSpecificInput(textEntityPairs, 2)
+      editDialogModal.VerifyEntityLabelWithinSpecificInput([textEntityPairs[0]], 0)
+      editDialogModal.VerifyEntityLabelWithinSpecificInput(textEntityPairs, 1)
+      editDialogModal.VerifyEntityLabelWithinSpecificInput(textEntityPairs, 2)
 
-    editDialogModal.RemoveEntityLabel('Tag', 'multi', 1)
-    editDialogModal.RemoveEntityLabel('Frog', 'multi', 2)
+      editDialogModal.RemoveEntityLabel('Tag', 'multi', 1)
+      editDialogModal.RemoveEntityLabel('Frog', 'multi', 2)
+    })
 
-    editDialogModal.ClickSubmitChangesButton()
-    editDialogModal.VerifyEntityLabeledDifferentPopupAndClose(textEntityPairs)
-    editDialogModal.ClickSubmitChangesButton()
-    editDialogModal.VerifyEntityLabeledDifferentPopupAndAccept(textEntityPairs)
+    
+    it('Should verify user cannot submit changes without accepting auto-re-labling of those two entity labels in the alternative input', () => {
+      editDialogModal.ClickSubmitChangesButton()
+      editDialogModal.VerifyEntityLabeledDifferentPopupAndClose(textEntityPairs)
+      editDialogModal.ClickSubmitChangesButton()
+      editDialogModal.VerifyEntityLabeledDifferentPopupAndAccept(textEntityPairs)
 
-    editDialogModal.VerifyEntityLabeledDifferentPopupAndClose(textEntityPairs)
-    editDialogModal.ClickSubmitChangesButton()
-    editDialogModal.VerifyEntityLabeledDifferentPopupAndAccept(textEntityPairs)
+      editDialogModal.VerifyEntityLabeledDifferentPopupAndClose(textEntityPairs)
```

Comment:
Why does this pop up again, if they accepted the changes in the above and it's the same `textEntityPairs` ?  I would expect the conflict to be resolved and the window should not pop up any more.

Reply:
In this test case we create two errors in two different alt-inputs. So after we fix one of them we have the other one popup. We first close it to confirm that we can not get away with not fixing it, then we follow up and accept it.